### ISSUE TITLE
HParams for training the model from readme

### ISSUE THREAD

**Issue by gsoul** [NVIDIA/tacotron2#55]
Hello

If that's possible, could you please kindly share the parameters that were used to train the model that produced the sample from the readme (ideally both Tacotron2 and WaveNet).

@rafaelvalle, you indicated here that this might be a possibility: https://github.com/NVIDIA/tacotron2/issues/31#issuecomment-400345054

Thank you in advance.

**Comment by rafaelvalle** [comment]
Yes, we will share the params and possibly a model checkpoint soon after we're done refactoring the model.

**Comment by gsoul** [comment]
If that's possible, could you please give a hint on when that might happen?

**Comment by rafaelvalle** [comment]
Probably this week.

**Comment by EthanKoo** [comment]
It's very kind, and expecting for the sharing ~

I have trained the model for 160K steps, [the wave](https://soundcloud.com/user-795890198/demo-160k) sounds good, but still far from the sample provided.

If possible, cloud you tell me how many steps have you trained for the model which generating the sample wave?

thx~

**Comment by rafaelvalle** [comment]
We're evaluating models that start learning attention as fast as 1500 iterations as one can see in the picture below. We're in the process of comparing these models to others.

![screen shot 2018-07-22 at 10 05 45 am](https://user-images.githubusercontent.com/2302607/43048095-e39afe88-8d96-11e8-9410-c48ca028af22.png)

**Comment by gsoul** [comment]
Hey, @rafaelvalle, if that's a possibility, could you please share the hparams that you used for training demo samples now, and then post the new code when it's ready. It would be extremely beneficial to the ones who try to reproduce your work with smaller hardware setups, as it takes ~2 weeks to train WaveNet to ~1mln steps...

**Comment by rafaelvalle** [comment]
We'll be releasing the new model and hprams in a branch soon.

**Comment by mikhail-barg** [comment]
Would be really nice to see actual hparams for the published sample.

We've tried running ttraining with default hparams on LJ dataset as explained in readme, and got incomparably worse results even after whole week of learning. Alignments seem to be OK, but the voice itself it very shaky, which doe not seem to be an effect of GL.

So we are not sure, is this because we did something wrong, or because we need to guess proper hparams, or something else
